queries.sql

```diff
@@ -175,4 +175,12 @@ INNER JOIN vets B ON V.vet_id = B.id
 INNER JOIN species S ON A.species_id = S.id
 WHERE B.name = 'Maisy Smith'
 GROUP BY S.name
-ORDER BY visits DESC LIMIT 1;
\ No newline at end of file
+ORDER BY visits DESC LIMIT 1;
+
+--Create queries to check performance
+
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits where animal_id = 4;
+
+EXPLAIN ANALYSE SELECT * FROM visits where vet_id = 2;
```

Comment:
- [ ] [optional] @team great work on your queries, even if the execution times difference is very small in your case, I can clearly see that you had some fun with the explain analyze and I am sure that in the future you will know how to start the investigation for some slow DB queries

Reply:
We really had so much fun. Thank you for that. :-)